framework/py/flwr/cli/run/run.py

```diff
@@ -100,27 +101,53 @@ def run(
     try:
         if suppress_output:
             redirect_output(captured_output)
+
+        original_app_str = str(app) if app is not None else ""
+        remote_app_ref: Optional[str] = None  # "user_name/app_name" if given with "@"
```

Comment:
we only use this variable as a boolean. Should we define it as such and set it to `True` if the app starts with `@`?